src/analysis/special_functions/trigonometric.lean

```diff
@@ -1759,4 +1761,126 @@ lemma differentiable_at_tan_of_mem_Ioo {x : ℝ} (h : x ∈ set.Ioo (-(π/2):ℝ
 lemma deriv_tan_of_mem_Ioo {x : ℝ} (h : x ∈ set.Ioo (-(π/2):ℝ) (π/2)) : deriv tan x = 1 / (cos x)^2 :=
 (has_deriv_at_tan_of_mem_Ioo h).deriv
 
+lemma continuous_on_tan_of_mem_Ioo : continuous_on tan (set.Ioo (-(π/2)) (π/2)) :=
+begin
+  refine continuous_on_tan.mono _,
+  intros x hx,
+  simp only [set.mem_set_of_eq],
+  exact ne_of_gt (cos_pos_of_mem_Ioo hx.1 hx.2),
+end
+
+open filter
+open_locale topological_space
+
+lemma tendsto_sin_pi_div_two : tendsto sin (𝓝[set.Iio (π/2)] (π/2)) (𝓝 1) :=
+by { convert (@continuous.continuous_within_at _ _ _ _ _ (set.Iio (π/2)) (π/2) continuous_sin).tendsto,
+  simp }
+
+lemma tendsto_cos_pi_div_two : tendsto cos (𝓝[set.Iio (π/2)] (π/2)) (𝓝[set.Ioi 0] 0) :=
+begin
+  change (tendsto cos (𝓝[set.Iio (π/2)] (π/2)) (_ ⊓ _)),
+  rw tendsto_inf,
+  split,
+  { convert (@continuous.continuous_within_at _ _ _ _ _ (set.Iio (π/2)) (π/2) continuous_cos).tendsto,
+    simp },
+  { rw tendsto_principal,
+    use set.Ioi (0:ℝ),
+    split,
+    { exact mem_nhds_sets is_open_Ioi (by simp [pi_div_two_pos]) },
+    { use set.Iio (π/2),
+      split,
+      { exact λ (x:ℝ), set.mem_Iio.mpr },
+      { intro x,
+        simp,
+        intros h1 h2,
+        exact cos_pos_of_mem_Ioo (by linarith) h2 } } },
+end
+
+lemma tendsto_tan_pi_div_two : tendsto tan (𝓝[set.Iio (π/2)] (π/2)) at_top :=
+begin
+  convert tendsto_mul_at_top (by norm_num) (tendsto.inv_tendsto_zero tendsto_cos_pi_div_two)
+            tendsto_sin_pi_div_two,
+  ext x,
+  ring,
+  apply tan_eq_sin_div_cos x,
+end
+
+lemma tendsto_sin_neg_pi_div_two : tendsto sin (𝓝[set.Ioi (-(π/2))] (-(π/2))) (𝓝 (-1)) :=
+by { convert (@continuous.continuous_within_at _ _ _ _ _ (set.Ioi (-(π/2))) (-(π/2)) continuous_sin).tendsto,
+  simp [sin_neg (π/2)] }
+
+lemma tendsto_cos_neg_pi_div_two : tendsto cos (𝓝[set.Ioi (-(π/2))] (-(π/2))) (𝓝[set.Ioi 0] 0) :=
+begin
+  change (tendsto cos (𝓝[set.Ioi (-(π/2))] (-(π/2))) (_ ⊓ _)),
+  rw tendsto_inf,
+  split,
+  { convert (@continuous.continuous_within_at _ _ _ _ _ (set.Ioi (-(π/2))) (-(π/2)) continuous_cos).tendsto,
+    simp [cos_neg (π/2)] },
+  { rw tendsto_principal,
+    use set.Iio (0:ℝ),
+    split,
+    { exact mem_nhds_sets is_open_Iio (by simp [pi_div_two_pos]) },
+    { use set.Ioi (-(π/2)),
+      split,
+      { exact λ (x:ℝ), set.mem_Ioi.mpr },
+      { intro x,
+        simp,
+        intros h1 h2,
+        exact cos_pos_of_mem_Ioo h2 (by linarith) } } },
+end
+
+lemma tendsto_tan_neg_pi_div_two : tendsto tan (𝓝[set.Ioi (-(π/2))] (-(π/2))) at_bot :=
+begin
+  convert tendsto_mul_at_bot (by norm_num) (tendsto.inv_tendsto_zero tendsto_cos_neg_pi_div_two)
+            tendsto_sin_neg_pi_div_two,
+  ext x,
+  ring,
+  apply tan_eq_sin_div_cos x,
+end
+
+/-!
+## Continuity and differentiability of arctan
+The continuity of `arctan` is difficult to prove due to `arctan` being (indirectly) defined naively
```

Comment:
```suggestion
### Continuity and differentiability of arctan

The continuity of `arctan` is difficult to prove due to `arctan` being (indirectly) defined naively
```
We use `###` for section headings like these. (It doesn't have to be fixed in this PR, but it looks like many of the other module docs in this file should be changed as well, e.g. the ones like ``/-! `complex.cosh`-/``.)